test/commands/block/get.test.js

```diff
@@ -115,5 +116,33 @@ describe('block:get', () => {
 				]);
 				return expect(printMethodStub).to.be.calledWithExactly(queryResult);
 			});
+
+		setupTest()
```

Comment:
Should we add a similar filter + test to accounts?

Reply:
yes! i will make it on another issue/PR